spec/services/x509/attribute_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+describe X509::Attribute do
+  let(:x509_subject) { 'O=US, OU=DoD, CN=John.Doe.1234' }
+
+  subject { described_class.new(raw: x509_subject) }
+
+  # rubocop:disable UnneededInterpolation
+  describe 'delegation' do
+    it 'delegates to raw' do
+      expect(subject.blank?).to eq false
+      expect(subject.present?).to eq true
+      expect(subject.to_s).to eq x509_subject
+      expect(subject.to_str).to eq x509_subject
+      expect("#{subject}").to eq x509_subject
```

Comment:
What is the purpose of testing this interpolation and how is it different from testing `subject.to_s`?

Reply:
Good question. I copied the tests for the `Pii::` equivalent. There shouldn't be a difference.